src/pages/offers/offer-review-list.tsx

```diff
@@ -0,0 +1,12 @@
+import { Reviews } from '../../types';
+import OfferReview from './offer-review';
+
+type ReviewProps = {
+  review: Reviews;
+};
+
+export default function OfferReviewList ({review}: ReviewProps): JSX.Element {
+  return (
+    <ul className="reviews__list">{review.map((currentReview) => OfferReview(currentReview))}</ul>
```

Comment:
И везде по коду, мы на реакте пишем, а не нативном JS, чтобы всё вызывать как функции

```suggestion
    <ul className="reviews__list">{review.map((currentReview) =><OfferReview currentReview />)}</ul>
```